### DISCUSSION TITLE
selecting multiple drawings

### DISCUSSION THREAD

**Discussion by waitingodot** [keiwando/toonsquid#276] [Q&A]
Brand new user here, so my issues may be dumb…

I’ve been animating in LumaFusion, and imported an animation loop sequence (32 individual drawings) into ToonSquid as an Animatioon Sequence. In the Timeline, I would like to copy and paste this loop numerous times, before applying a Transformation to the entire sequence.

The TS Handbook says you can select multiple drawings in the Timeline by double tapping and dragging a selection window across the drawings. I haven’t had any success; no selection window opens. 

Also, I found the option for Duplicate, which adds a duplicate Animation Sequence ABOVE the original sequence. But what I’d like to do is paste a duplicate copy of the full Sequence (32 drawings) to the END of the existing sequence.

The number of frames per drawing within the loop is idiosyncratic, so I want to be able to copy the drawings and their timings, and duplicate them.

This is all relatively easy to accomplish in LumaFusion, but I haven’t figured it out in ToonSquid. 

I’m eager to use a Transform Layer to scale and position the Sequence, while maintaining vector resolution, and using Keyframe Easing. These features are NOT possible in LumaFusion. 

Thanks for any help

**Comment by Cat-Butler** [comment]
What you need to do is double-tap-drag without releasing after the second tap, it is a little tricky to first get the hang of this, it's a tap then tap-drag. If you release after the double-tap it does not work.
The "selection box" is just a blue semi-transparent highlight rectangle created by the dragging part.

https://user-images.githubusercontent.com/25494772/233528544-159c2c93-63f0-466e-91e8-501f0774d338.mov

If you are importing an animation sequence, which will currently only be imported as raster/pixel content, you will not get vector resolution when transforming / scaling that imported content. Currently you will only get vector resolution transformation / scaling for new vector content, that is vector shapes/curves and layers painted with vector brushes, created within the ToonSquid project.

Importing external vector content (for example, SVG format) has been requested as a new feature by some users.
